x-pack/plugins/cloud_security_posture/server/routes/status/status.test.ts

```diff
@@ -113,7 +185,7 @@ describe('CspSetupStatus route', () => {
     const body = call[0]?.body;
     expect(mockResponse.ok).toHaveBeenCalledTimes(1);
 
-    await expect(body).toEqual({
+    await expect(body).toMatchObject({
```

Comment:
looks like `toMatchObject` is a more loose version of `toEqual`. seems like we'd prefer the stricter one. can you clarify this change ?

Reply:
yep, to ease maintenance, whenever you add new fields you should add proper tests for it. instead of now maintaining 10+ tests whenever you change something.

That's the idea behind using `toMatchObject` here

wdyt?